app/src/main/java/com/berniesanders/fieldthebern/screens/Main.java

```diff
@@ -62,6 +63,16 @@ public String getScopeName() {
     return Main.class.getName();
   }
 
+  public static final Creator<Main> CREATOR = new ScreenCreator<Main>() {
+    @Override protected Main doCreateFromParcel(Parcel source) {
```

Comment:
I think the idea is that the Screens are value objects that represent an entry in the backstack which Flow will parcel to save the history.  I'd say the easiest thing to do would be to set the field directly here when recreating the object from its parcel.  Since we've basically delegated the wrapping/unwrapping of a parcel to each screen.  So in this case something like: 
```java
@Override 
protected Main doCreateFromParcel(Parcel source) {
  Main main = new Main();
  main.savedState = source.readParcelable(Main.class.getClassLoader());
  return main;
}
```
I'm not 1000% sure on all this, but I think this is what we want to do.